client/questions/expandAnswers.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 
 
-const ExpandAnswers = (props) => {
-  if (props.answersList.length > 2 && props.show) {
+const ExpandAnswers = ({answers, answersList, show, seeMoreAnswers}) => {
+
+  if (answersList.length > 2 && show) {
     return (
-      <p className="expand-answers-btn" onClick={() => props.seeMoreAnswers()}>See More Answers</p>
+      <p className="expand-answers-btn" onClick={() => seeMoreAnswers()}>See More Answers</p>
     )
   } else {
-    return null;
+    if (answersList.length === answers.length && answersList.length > 2) {
+      return (
+        <p className="expand-answers-btn" onClick={() => seeMoreAnswers()}>Collapse answers</p>
```

Comment:
"onClick={() => seeMoreAnswers()}" can be simplified as "onClick={ seeMoreAnswers }"